cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -33,6 +33,12 @@ resourcesCrossplane:
     cpu: 100m
     memory: 256Mi
 
+securityContextCrossplane:
+  runAsUser: 2000
+  runAsGroup: 2000
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
```

Comment:
@khos2ow could you break these parameters out into a separate PR? I would definitely like for these values to be configurable such that crossplane can run on platforms that restrict setting them, but would like for the change to be tracked as a distinct PR :)

Reply:
Sure, #1983 should do it. I'm gonna wait for that to land and then rebase it here.